### ISSUE TITLE
problem when running run.py

### ISSUE THREAD

**Issue by 94happy** [qq456cvb/VoteNet#3]
My environment is : python3.6 + cuda9.0 +tensorflow1.8.0, and  I want to know what's your environment?
when I run "python run.py", the error is the following:

[0521 15:58:12 @base.py:236] Initializing the session ...
[0521 15:58:12 @base.py:243] Graph Finalized.
[0521 15:58:12 @concurrency.py:38] Starting EnqueueThread QueueInput/input_queue ...
*** Received signal 11 ***
*** BEGIN MANGLED STACK TRACE ***
/home/liuzhe/anaconda3/lib/python3.6/site-packages/tensorflow/python/../libtensorflow_framework.so(+0x5982fb)[0x7effe6d792fb]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x11390)[0x7f0020be7390]
/home/liuzhe/deep_learning_code/VoteNet/tf_ops/3d_nms/tf_nms3d_so.so(+0x6a1e)[0x7eff5b3aba1e]
/home/liuzhe/deep_learning_code/VoteNet/tf_ops/3d_nms/tf_nms3d_so.so(_ZNSt17_Function_handlerIFbiiiESt5_BindIFPFfN5Eigen9TensorMapINS2_6TensorIKfLi4ELi1ElEELi16ENS2_11MakePointerEEEiiiifES8_St12_PlaceholderILi1EESB_ILi2EESB_ILi3EEifEEE9_M_invokeERKSt9_Any_dataOiSL_SL_+0x30)[0x7eff5b3aacb0]
/home/liuzhe/deep_learning_code/VoteNet/tf_ops/3d_nms/tf_nms3d_so.so(_Z21DoNonMaxSuppressionOpPN10tensorflow15OpKernelContextERKNS_6TensorES4_iifSt8functionIFbiiiEE+0x940)[0x7eff5b3aa4a0]
/home/liuzhe/deep_learning_code/VoteNet/tf_ops/3d_nms/tf_nms3d_so.so(_ZN21NonMaxSuppression3DOpIN5Eigen16ThreadPoolDeviceEE7ComputeEPN10tensorflow15OpKernelContextE+0x3b7)[0x7eff5b3ad417]
/home/liuzhe/anaconda3/lib/python3.6/site-packages/tensorflow/python/../libtensorflow_framework.so(_ZN10tensorflow16ThreadPoolDevice7ComputeEPNS_8OpKernelEPNS_15OpKernelContextE+0x14c)[0x7effe712150c]
/home/liuzhe/anaconda3/lib/python3.6/site-packages/tensorflow/python/../libtensorflow_framework.so(+0x90624d)[0x7effe70e724d]
/home/liuzhe/anaconda3/lib/python3.6/site-packages/tensorflow/python/../libtensorflow_framework.so(+0x906a7a)[0x7effe70e7a7a]
/home/liuzhe/anaconda3/lib/python3.6/site-packages/tensorflow/python/../libtensorflow_framework.so(_ZN5Eigen26NonBlockingThreadPoolTemplIN10tensorflow6thread16EigenEnvironmentEE10WorkerLoopEi+0x21a)[0x7effe6d4d6aa]
/home/liuzhe/anaconda3/lib/python3.6/site-packages/tensorflow/python/../libtensorflow_framework.so(_ZNSt17_Function_handlerIFvvEZN10tensorflow6thread16EigenEnvironment12CreateThreadESt8functionIS0_EEUlvE_E9_M_invokeERKSt9_Any_data+0x32)[0x7effe6d4c752]
/home/liuzhe/anaconda3/bin/../lib/libstdc++.so.6(+0xafc5c)[0x7f001c2e2c5c]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x76ba)[0x7f0020bdd6ba]
/lib/x86_64-linux-gnu/libc.so.6(clone+0x6d)[0x7f002091341d]
*** END MANGLED STACK TRACE ***

*** Begin stack trace ***
	tensorflow::CurrentStackTrace()
	
	
	
	std::_Function_handler<bool (int, int, int), std::_Bind<float (*(Eigen::TensorMap<Eigen::Tensor<float const, 4, 1, long>, 16, Eigen::MakePointer>, std::_Placeholder<1>, std::_Placeholder<2>, std::_Placeholder<3>, int, float))(Eigen::TensorMap<Eigen::Tensor<float const, 4, 1, long>, 16, Eigen::MakePointer>, int, int, int, int, float)> >::_M_invoke(std::_Any_data const&, int&&, int&&, int&&)
	DoNonMaxSuppressionOp(tensorflow::OpKernelContext*, tensorflow::Tensor const&, tensorflow::Tensor const&, int, int, float, std::function<bool (int, int, int)>)
	NonMaxSuppression3DOp<Eigen::ThreadPoolDevice>::Compute(tensorflow::OpKernelContext*)
	tensorflow::ThreadPoolDevice::Compute(tensorflow::OpKernel*, tensorflow::OpKernelContext*)
	
	
	Eigen::NonBlockingThreadPoolTempl<tensorflow::thread::EigenEnvironment>::WorkerLoop(int)
	std::_Function_handler<void (), tensorflow::thread::EigenEnvironment::CreateThread(std::function<void ()>)::{lambda()#1}>::_M_invoke(std::_Any_data const&)
	
	
	clone
*** End stack trace ***
Aborted (core dumped)

Can you help me? thanks!

**Comment by qq456cvb** [comment]
Do you pull the latest source code? What if you run 'tf_nms3d.py'? This should print out an array with no errors.

**Comment by 94happy** [comment]
Yes, thanks for your reply! I run "tf_nms3d.py" and it can print out an array. But it  still occurs this same errors! 
![Selection_033](https://user-images.githubusercontent.com/28643323/58087943-38ff4180-7bf4-11e9-86f0-06a8f69ea534.png)

**Comment by 94happy** [comment]
My environment is : python3.6 + cuda9.0 +tensorflow1.8.0 + tensorpack0.9.4 . Can you  provide me your version of tensorflow-gpu , tensorpack, and the output  log file in training process ?  

Sometimes I run the code and it stays here all the time:
![Selection_032](https://user-images.githubusercontent.com/28643323/58087802-e3c33000-7bf3-11e9-93af-4229532efd44.png)

**Comment by qq456cvb** [comment]
My environment is Python 3.5.2 + cuda 10.0 + TensorFlow 1.13.1 + Tensorpack 0.9.1. It seems the code gets stuck but it is actually evaluating mAP on validation set. I have made a commit to show the progress bar when evaluating mAP and you can now see a progress bar without 'stuck'. Also, you may tune this feature by modifying callbacks in 'run.py', see https://tensorpack.readthedocs.io/modules/callbacks.html#tensorpack.callbacks.PeriodicTrigger.

**Comment by 94happy** [comment]
It works! Thank you very much for your patience!
